### DISCUSSION TITLE
add xapp to status bar

### DISCUSSION THREAD

**Discussion by ghost** [i3/i3#5863] [Q&A]
I recently installed i3 on my linux mint system. I recently got a notification to add xapp. Would it be the same as adding a custom browser? Or is there another way I should do it. The only thing I found on the internet about adding it was for a full desktop environment.

**Comment by orestisfl** [comment]
What is xapp exactly?

**Comment by ghost** [comment]
it allows some gtk applications to be able to be added to the notification area of the panel on linux mint.
